packages/devtools_app/lib/src/analytics/analytics_common.dart

```diff
@@ -0,0 +1,9 @@
+// Copyright 2021 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// Code in this file should be able to be imported by both dart:html and
+// dart:io dependent libraries.
+
+/// Base class for all screen metrics classes.
```

Comment:
Describe what is involved in creating a subclass or switch to a 
toJson() based scheme so that it isn't magical.

Reply:
done